### DISCUSSION TITLE
1.0.9.6

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#75] [Обновления]
# 1.0.9.6
- Теперь, слайдер громкости оригинала появляется, даже, если перевод выключен
- Теперь, слайдер громкости оригинала появляется и убирается простым нажатием на галочку 
- Добавлена синхронизация слайдера громкости оригинала со слайдером громкости видео (пока только для YouTube (именно со слайдером громкости, а не с громкостью видео - сделано так из-за специфики ютуба)
- Добавлена опциональная синхронизация громкости перевода с громкостью видео (пока только для YouTube)

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.0.9.6'>1.0.9.6</a>.</em>

**Comment by Dionis94** [comment]
Спасибо за добавление по процентам громкости оригинала. Очень спасает когда надо тихий звук настроить. Спасибо!!!

**Reply by kuplas** [reply]
Планируется ли добавление субтитров? Или возможна только озвучка?

**Reply by ilyhalight** [reply]
> Планируется ли добавление субтитров? Или возможна только озвучка?

Возможна только озвучка.

**Comment by FiLinX** [comment]
Скажите, а как мне откатиться на 1.0.9.4 ?
А то в нынешней версии не работает на эмбед ютубинах, и для меня это считай что переводилка совсем не работает, так как 95% роликов просматриваю лишь в эмбед.

вроде откатился.. перевод вновь работает!  я доволен)

**Comment by nicola-sh** [comment]
А как правильно добавить сайт udemy.com в скрипт, дабы видеоуроки с этого сайта переводились? там имеются субтитры на английском уж точно
